Week2/francisco-torres/spring-rest/src/main/java/com/globant/bootcamp/springrest/controller/DateController.java

```diff
@@ -0,0 +1,29 @@
+package com.globant.bootcamp.springrest.controller;
+
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDate;
+
+@RestController
+@RequestMapping("/date")
```

Comment:
The resources names usually should be plural, like /dates, because they are collections.